README.md

```diff
@@ -1,7 +1,7 @@
-# mkosi - Create legacy-free OS images
+# mkosi - Create bespoke OS images
```

Comment:
maybe change this "Build Bespoke OS Image"? i.e. use uppercase since we use it almost everywhere else, and "build" instead of "create" since it's really about building and is what we use at most places already